.github/workflows/build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             - run: sudo apt install libwayland-dev libxrandr-dev
 
             - name: Get Google Test
-              run: git clone --branch release-1.10.0 https://github.com/google/googletest.git external/googletest
```

Comment:
Thinking about it, with the loader when using UPDATE_DEPS for getting these dependencies, these steps in the yml are redundant.

Reply:
REmoved.